app/scripts/controllers/network/createInfuraClient.js

```diff
@@ -6,7 +6,7 @@ import {
   createInflightCacheMiddleware,
   createBlockTrackerInspectorMiddleware,
   providerFromMiddleware,
-} from 'eth-json-rpc-middleware';
```

Comment:
Hmm, it should be removed already!

Reply:
sorry for coming back late to this one, i was asking this because i still see the library in the dependency list, see https://github.com/MetaMask/metamask-extension/blob/develop/package.json#L286. 
but i think we can also move it in another PR to be safe.

Reply:
Ah, but it is removed in this PR: https://github.com/MetaMask/metamask-extension/pull/17591/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Reply:
Ah sorry just saw  it now was missed, thanks for pointing out ~